core/src/test/java/io/grpc/internal/ServerCallImplCommonTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015, Google Inc. All rights reserved.
+ * Copyright 2017, Google Inc. All rights reserved.
```

Comment:
Don't update the dates. Updating to 2017 isn't right. If anything, this would be `2015-2017`. But it isn't worth our time. Just make sure that new files have the right date.